pages/common/jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+> Open source java application designed for load testing functional behavior and measure performance.
+> More information: <https://jmeter.apache.org>.
+
+- Run a specific test plan in nongui mode:
+
+`jmeter --nongui --testfile {{path/to/file}}.jmx`
+
+- Run a test plan in nongui mode using a specific log file:
+
+`jmeter --nogui --testfile {{path/to/file}}.jmx --logfile {{path/to/logfile}}.jtl`
+
+- Use a specific proxy server and a specific port:
+
+`jmeter --nongui --testfile {{path/to/file}}.jmx --proxyHost {{127.0.0.1}} --proxyPort {{8888}}`
+
+- Define user variables:
```

Comment:
The man page <https://manned.org/jmeter> refers to this variables as `JMeter properties` so:
```suggestion
- Run a test plan in nongui mode using a specific JMeter property:
```